tests/unit/plugins/modules/test_kubevirt_vm.py

```diff
@@ -169,6 +179,18 @@ def k8s_module_params_create(module_params_create, vm_manifest):
     }
 
 
+@pytest.fixture(scope="module")
+def k8s_module_params_create(k8s_module_params_common, module_params_create):
+    return module_params_create | k8s_module_params_common
+
+
+@pytest.fixture(scope="module")
+def k8s_module_params_delete(k8s_module_params_common, module_params_delete):
+    return module_params_delete | k8s_module_params_common | {
+        "wait": True,
```

Comment:
Duplicate param, already defined in module_params_delete?

Reply:
Yes, but because in `k8s_module_params_common` it is set to `False` and `k8s_module_params_common` was after `module_params_delete`, the `wait` field was replaced from `True` to `False`. I've inverted the order to pick the latest fields on `module_params_delete`. Fixed.